StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -604,13 +606,15 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
 
     func disableNetworking(
         disabledReason: String?,
+        clientSuggestedNextPaneOnDisableNetworking: String?,
         clientSecret: String
     ) -> Future<FinancialConnectionsSessionManifest> {
         var body: [String: Any] = [
             "client_secret": clientSecret,
             "expand": ["active_auth_session"],
         ]
         body["disabled_reason"] = disabledReason
+        body["client_requested_next_pane_on_disable_networking"] = clientSuggestedNextPaneOnDisableNetworking
```

Comment:
[stripe.js](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/pull/23246/files#diff-ee09cc629c5030b4f8cdb14513bc50a01d0e24dac4cf15d7073aac9a58e941b3R132-R143)

Reply:
this is the core change that activates "Not Now" from manual entry button to keep going to manual entry (instead of institution picker)